src/components/SignUpForm.jsx

```diff
@@ -14,6 +14,14 @@ const CreateAccount = (props, routeProps) => {
 		setFieldType(fieldType === 'Password' ? 'text' : 'Password');
 	};
 
+	if(formIncomplete(props) === true || props.values.ContactID === null){
```

Comment:
looks like this same exact code is repeated in Sign In Form, please consolidate these.

Reply:
code had to be changed to reflect logic that works. I wont have a user ID if I haven't logged in or created an account yet soooo this no workey anyway. Its been fixed.